exercises/simple-linked-list/simple-linked-list.spec.js

```diff
@@ -49,7 +49,7 @@ describe('simple-linked-list', function () {
 
   xit('requires an instance of Element as next element', function () {
     expect(function () {
-      var el = new Element(1, true);
+      // var el = new Element(1, true);
```

Comment:
Please remove commented out code from the PR.

Reply:
>  Why did you leave the commented out code?

Sorry, I left the commented out code in there because I wasn't sure what the original dev was intending with making the same constructor call with true and false boolean values. I've removed it. Will push this alongwith the other changes you requested in a new PR as soon as I get some clarification on the comments I made above.

Reply:
@natraj-s please don't open a new PR! You can just make the changes on the same branch and push it out to your repo.  The PR will update.

Reply:
lol, sorry, I meant to say I'll commit my changes, not open a new PR.

Reply:
OK great.  You would be surprised how many people close and then open a new one 👍